tests/framework/defs.py

```diff
@@ -57,10 +45,9 @@
 
 SUPPORTED_HOST_KERNELS = ["4.14", "5.10", "6.1"]
 
-SUPPORTED_KERNELS = ["4.14", "5.10"]
-SUPPORTED_KERNELS_NO_SVE = ["4.14", "5.10-no-sve"]
-
+SUPPORTED_KERNELS = [r"vmlinux-4.14.\d+", r"vmlinux-5.10.\d+"]
+SUPPORTED_KERNELS_NO_SVE = [r"vmlinux-4.14.\d+", r"vmlinux-5.10.\d+-no-sve"]
 
-def _test_images_s3_bucket():
-    """Auxiliary function for getting this session's bucket name."""
-    return os.environ.get(ENV_TEST_IMAGES_S3_BUCKET, DEFAULT_TEST_IMAGES_S3_BUCKET)
+ARTIFACT_DIR = (
+    Path(__file__).parent.joinpath("../../build/img").resolve() / platform.machine()
```

Comment:
nit: mh, we're mixing `.parent` and `../..` here. Can we just do `Path(__file__).joinpath("../../../build/img").resolve() / platform.machine()`?